.nycrc

```diff
@@ -1,3 +1,10 @@
 {
-  "extends": "node_modules/ocular-dev-tools/templates/.nycrc"
+  "extends": "node_modules/ocular-dev-tools/templates/.nycrc",
+  "include": [
+    "dev-modules/**/*.js",
```

Comment:
Nit: Worth avoiding different directory structures between modules and dev-modules? 

Even if dev-modules often just have one source file, just keep the same structure to follow the principle of least surprises, and so we can move them between different `modules` and `dev-modules` directories in the future?